source/features/branch-buttons.tsx

```diff
@@ -74,9 +72,7 @@ async function getDefaultBranchLink(): Promise<HTMLElement | undefined> {
 	if (isRepoRoot()) {
 		url = `/${getRepoURL()}`;
 	} else {
-		const urlParts = location.pathname.split('/');
-		urlParts[4] = defaultBranch; // Change branch of current blob/tree
-		url = urlParts.join('/');
+		url = location.pathname.replace(currentBranch, defaultBranch);
```

Comment:
This isn't safe, it will fail on `/sindresorhus/ava/blob/ava`

Reply:
You're right! I forgot about that, thx!